guides/06_FAQs.md

```diff
@@ -0,0 +1,239 @@
+
+## Debugging
+
+### How can I get Amazon Chime SDK logs for debugging?
+
+Applications can get logs from Chime SDK by passing instances of Logger when instantiating [the MeetingSession object](https://github.com/aws/amazon-chime-sdk-js/blob/master/src/meetingsession/MeetingSession.ts#L14). Amazon Chime SDK has some default implementations of logger that your application can use, such as [ConsoleLogger](https://github.com/aws/amazon-chime-sdk-js/blob/8e5802c04031ac62a2c5c5fe75af5bb4ffbf12d0/src/logger/ConsoleLogger.ts) which logs into the browser console, [MeetingSessionPOSTLogger](https://github.com/aws/amazon-chime-sdk-js/blob/8e5802c04031ac62a2c5c5fe75af5bb4ffbf12d0/src/logger/MeetingSessionPOSTLogger.ts) which logs in [Amazon CloudWatch](https://aws.amazon.com/cloudwatch/) and [MultiLogger](https://github.com/aws/amazon-chime-sdk-js/blob/8e5802c04031ac62a2c5c5fe75af5bb4ffbf12d0/src/logger/MultiLogger.ts) which logs in multiple destinations.
+
+On mobile phones such as iPhone you can also do some advance debugging by connecting the phone to a Mac such as highlighted in this [article](https://developer.apple.com/library/archive/documentation/AppleApplications/Conceptual/Safari_Developer_Guide/GettingStarted/GettingStarted.html).
+
+### How do I file an issue for the Amazon Chime SDK for JavaScript?
+
+You can use our [bug template](https://github.com/aws/amazon-chime-sdk-js/issues/new?assignees=&labels=bug&template=bug-report.md&title=) to file issues with logs (it is helpful to set the logging level as INFO) and exact reproduction steps. To help you faster, you can check the usage of the API in our [API overview](https://aws.github.io/amazon-chime-sdk-js/modules/apioverview.html), [demos](https://github.com/aws/amazon-chime-sdk-js/blob/master/README.md#examples) and the [usage section](https://github.com/aws/amazon-chime-sdk-js/blob/master/README.md#usage) in our Readme. In addition search our [issues database](https://github.com/aws/amazon-chime-sdk-js/issues?q=is:issue) as your concern may have been addressed previously and mitigations may have been posted.
+
+## Networking
+
+### What are the subnets and ports used by an application using the AWS SDK?
+
+An application which calls Amazon Chime APIs via the [AWS SDK](https://aws.amazon.com/sdk-for-node-js/), like [CreateMeeting](https://docs.aws.amazon.com/chime/latest/APIReference/API_CreateMeeting.html), connects to the global service endpoint in the us-east-1 region: [service.chime.aws.amazon.com](http://service.chime.aws.amazon.com/), or [service-fips.chime.aws.amazon.com](http://service.chime.aws.amazon.com/) with TCP:443.
```

Comment:
nit: You have linked "AWS SDK for Node.js" but a customer can be using AWS SDK in any language. How about adding https://aws.amazon.com/tools ?